debian/control

```diff
@@ -8,6 +8,6 @@ Standards-Version: 1.0.1
 
 Package: libopx-sai-vm1
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, libopx-sai-common-utils1
+Depends: ${shlibs:Depends}, ${misc:Depends}, libopx-sai-common-utils1, libopx-db-sql1
```

Comment:
This shouldn't be necessary (neither should libopx-sai-common-utils1) as shlibs:Depends should contain libopx-db-sql1 in it's expansion.  Please check before adding.